### PR TITLE
Fix constraint motors moving in wrong direction

### PR DESCRIPTION
**Pull request by AlgoryxSofi**
When importing Brick file, motors on constraints seemed to move in the opposite direction of what you could see when loading the file outside of Unity. The reason was that the attachments (and bodies) chosen for reference frame and connecting frame in the constraint in AGX for Unity was wrong. Flipping which was which seems to have solved the problem.

### CODE REVIEW

File: Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs [hunk 1]
```diff
@@ -361,22 +361,23 @@ private GameObject HandleConnector(B_Connector b_connector, GameObject go_parent
       var b_body1 = b_attachment1.Body;
       var b_body2 = b_attachment2.Body;
       var c_attachmentPair = constraint.AttachmentPair;
-      if (b_body1 == null)
+      if (b_body2 == null)
       {
-        c_attachmentPair.ReferenceObject = bodyDict[b_body2 as B_RigidBody].gameObject;
+        c_attachmentPair.ReferenceObject = bodyDict[b_body1 as B_RigidBody].gameObject;
         c_attachmentPair.ReferenceFrame.LocalPosition = b_attachment2.LocalPosition.ToHandedVector3();
```
Comment (AlgoryxSandra) [inline Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs:367]: Should this be `b_attachment1`?

Comment (AlgoryxMartinT) [reply Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs:367]: Yes, good that you saw that!

Comment (AlgoryxSofi) [reply Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs:367]: Yes, good that you saw that!

Comment (AlgoryxSofi) [reply Editor/AGXUnityEditor/BrickUnity/BrickPrefabImporter.cs]: It should be fixed now

**Pull request merged**